monitoring/src/kieker/monitoring/probe/aspectj/operationExecution/AbstractOperationExecutionAspect.java

```diff
@@ -117,9 +115,12 @@ public void afterOperation(final JoinPoint thisJoinPoint) {
 		final OperationStartData data = this.stack.get().pop();
 
 		final long tout = TIME.getTime();
+		
+		final String hostname = VMNAME;
```

Comment:
Is this a static hostname or just a default?

Reply:
Its just the hostname, so its required here (and also present in AspectJ).

Reply:
So you moved sessionId and hostname out of the OperationStartData structure and ask that below prior to sending the record. Saves one assignment, so I am fine with it.